### PR TITLE
Update redirects.yaml

### PR DESCRIPTION
**Pull request by carkod**
## Done

[List of work items including drive-bys]

## QA

https://docs-ubuntu-com-355.demos.haus/landscape/en/ should redirect to https://ubuntu.com/landscape/docs

Test that these links redirect correctly

https://docs-ubuntu-com-355.demos.haus/landscape/en/concepts → https://ubuntu.com/landscape/docs

https://docs-ubuntu-com-355.demos.haus/landscape/en/onprem → https://ubuntu.com/landscape/docs/self-hosted-landscape 

https://docs-ubuntu-com-355.demos.haus/landscape/en/landscape-install-quickstart  → https://ubuntu.com/landscape/docs/quickstart-deployment 

https://docs-ubuntu-com-355.demos.haus/landscape/en/landscape-install-manual → https://ubuntu.com/landscape/docs/manual-installation 

https://docs-ubuntu-com-355.demos.haus/landscape/en/onprem-backup → https://ubuntu.com/landscape/docs/backup-and-restore 

https://docs-ubuntu-com-355.demos.haus/landscape/en/onprem-logs → https://ubuntu.com/landscape/docs/logs 

https://docs-ubuntu-com-355.demos.haus/landscape/en/onprem-auth → https://ubuntu.com/landscape/docs/external-authentication 

https://docs-ubuntu-com-355.demos.haus/landscape/en/api-client-package → https://ubuntu.com/landscape/docs/command-line-client 

https://docs-ubuntu-com-355.demos.haus/landscape/en/api-http-requests → https://ubuntu.com/landscape/docs/low-level-http-requests 

https://docs-ubuntu-com-355.demos.haus/landscape/en/api-methods → https://ubuntu.com/landscape/docs/api-overview

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-2513?atlOrigin=eyJpIjoiYTIwOTg1YmEyZGUyNDZlYTk5OWM1YWViYTgyYzBhYzUiLCJwIjoiaiJ9

### CODE REVIEW

File: redirects.yaml [hunk 1]
```diff
@@ -14,8 +14,7 @@ snap-enterprise-proxy: /snap-store-proxy/en/
 snap-enterprise-proxy/(?P<page>.*): /snap-store-proxy/{page}
 
 # Landsacpe
-landscape/en/landscape-api/?: /landscape/en/api
```
Comment (tbille) [inline redirects.yaml:17]: This is no 404ing with the new change. This redirect should be kept

File: redirects.yaml [hunk 2]
```diff
@@ -14,8 +14,7 @@ snap-enterprise-proxy: /snap-store-proxy/en/
...
-landscape/en/onprem-overview/?: /landscape/en/onprem
```
Comment (tbille) [inline redirects.yaml:18]: This redirect is missing:
```
https://docs.ubuntu.com/landscape/en/onprem → https://ubuntu.com/landscape/docs/self-hosted-landscape
```

File: redirects.yaml [hunk 3]
```diff
@@ -14,8 +14,7 @@ snap-enterprise-proxy: /snap-store-proxy/en/
 snap-enterprise-proxy/(?P<page>.*): /snap-store-proxy/{page}
 
 # Landsacpe
```
Comment (tbille) [inline redirects.yaml:16]: Can you fix the typo while you are here? :sweat_smile:

File: redirects.yaml (hunk 1 above)
Comment (carkod) [reply redirects.yaml:17]: I've actually added it to the [redirects table](https://discourse.ubuntu.com/t/landscape-documentation-home/23070?_ga=2.192479466.1109813418.1679260968-346780024.1629501030).

But it looks like it's not working, so I'll update the PR

**Comment by mtruj013** [comment]
Are you still working on this @carkod? Looks like the list of redirects Rajan left on the ticket have to be addressed still

**Comment by carkod** [comment]
> Are you still working on this @carkod? Looks like the list of redirects Rajan left on the ticket have to be addressed still

All Done.

**Review by mtruj013 — APPROVED** [review]
LGTM!

**Pull request merged**